backend/tracim_backend/views/frontend.py

```diff
@@ -17,15 +18,29 @@
 
 
 class FrontendController(Controller):
-    def __init__(self, dist_folder_path: str):
+    def __init__(self, dist_folder_path: str, custom_toolbox_folder_path: typing.Optional[str]):
         self.dist_folder_path = dist_folder_path
+        self.custom_toolbox_folder_path = custom_toolbox_folder_path
+        self.custom_toolbox_files = []  # typing.List["os.DirEntry"]
+        if custom_toolbox_folder_path:
+            self.custom_toolbox_files = self._get_custom_toolboxes_files(
+                self.custom_toolbox_folder_path
+            )
 
-    def _get_index_file_path(self):
+    def _get_index_file_path(self) -> str:
         index_file_path = os.path.join(self.dist_folder_path, INDEX_PAGE_NAME)
         if not os.path.exists(index_file_path):
             raise FileNotFoundError()
         return index_file_path
 
+    def _get_custom_toolboxes_files(self, custom_toolbox_dir: str) -> typing.List["os.DirEntry"]:
```

Comment:
```suggestion
    def _get_custom_toolboxes_files(self, custom_toolbox_dir: str) -> typing.List[os.DirEntry]:
```

Reply:
Not possible here. os.DirEntry is not importable in python 3.5. So if we do this, we have to import os.DirEntry and code will failed in 3.5.
see : https://stackoverflow.com/questions/38307995/create-os-direntry